QNormalizModule.cpp

```diff
@@ -381,7 +374,7 @@ PyObject* NmzToPyNumber( renf_elem_class in ){
     fmpq_poly_init(current);
     in.get_fmpq_poly(current);
     vector<mpq_class> output;
-    fmpq_poly2vector(output,current);
+    fmpq_poly2vector(output,current); // is this the correct length??
```

Comment:
I do not understand the length comment? There is no length given here. The function itself comes from e-antic.

Reply:
Sorry for being cryptic. Question is, do we want to represent number field elements by list of exactly the degree, or list of at most the degree of the number field (when some highest coefficients are 0).

Reply:
Good question, though. I personally to not care to much. The current solution is easy though, as the conversion function exists. one could for example resize the vector using the element output handler function.

Reply:
I'm handling the current format, of course, in sage, so no change is needed. But I was surprised first.